Gopkg.toml

```diff
@@ -62,8 +62,9 @@
   name = "github.com/juju/bundlechanges"
 
 [[constraint]]
-  revision = "7eee51ddb1f5033244c6501ec83b399af9d23861"
+  revision = "be85422a108107ae554950534717443e97dc70d0"
   name = "github.com/juju/description"
+  source = "github.com/SimonRichardson/description"
```

Comment:
Once this PR has been approved from a code point of view and it handles the descriptions correctly, I'll merge the description package first, before merging this PR.

The should ensure that the churn on description is limited.